src/OpenTelemetry.Exporter.InMemory/InMemoryExporterLoggingExtensions.cs

```diff
@@ -42,9 +42,17 @@ private static ExportResult ExportLogRecord(in Batch<LogRecord> batch, ICollecti
 
             foreach (var log in batch)
             {
-                log.BufferLogScopes();
+                log.Buffer();
 
-                exportedItems.Add(log);
+                LogRecord copy = new()
```

Comment:
consider moving this to the LogRecord class.
I think it would be easier to maintain a copy ctor.
`internal LogRecord(LogRecord other)`

Reply:
I refactored it into `class LogRecord { internal LogRecord Copy() {} }`. In-memory exporter has to see SDK internals to invoke the method but we already had that in place 😄 LMK what you think about that.